CMakeLists.txt

```diff
@@ -370,6 +372,44 @@ if(WITH_DLT_LOG_LEVEL_APP_CONFIG)
     add_definitions(-DDLT_LOG_LEVEL_APP_CONFIG)
 endif()
 
+
+if(WITH_DLT_TRACE_LOAD_CTRL)
+    add_definitions(-DDLT_TRACE_LOAD_CTRL_ENABLE)
+
+    # Configure limits for client
+    if(NOT TRACE_LOAD_USER_SOFT_LIMIT)
+        set(TRACE_LOAD_USER_SOFT_LIMIT 83333)
```

Comment:
Is there a specific reason to set the default soft limit to 83333 and the hard limit to 100000? 
Also, what is the maximum configurable soft and hard limit for my application?

Reply:
There are no specific reasons for the values. Internally we set the cmake parameters to 0 to force devs to create limits but I didn't think this was appropriate for upstream.

The maximum is UINT32_MAX for soft and hard. But that's a good point and should be documented

Reply:
This is the expectation from @michael-methner point of view for a general feature upstreaming. About why the defaults are chosen like this, idk, may a clearer explanation needed at this point @alexmohr ?

Reply:
I set the defaults internally to the same values, but this was quite some time ago and I can't remember the exact reason for this. iirc these where the defaults used by the previous system generation. I'm not attached to these values and if you want to make a suggestion, I'll change them :) (although I don't think the exact values are not that important as it can be configured)